include/model_mgmt.h

```diff
@@ -61,7 +61,8 @@ enum class model_type_t
   CB,
   CCB,
   SLATES,
-  CA
+  CA,
+  EPISODIC
```

Comment:
there isn't an episodic model type, it gets translated to cb

Reply:
this can be reverted, we're holding off on episodic for now

Reply:
yea I agree, there should be a separate concept of `loop_type` and `model_type`, since the two can be used interchangeably for the other loop/model types